examples/vite-react/package.json

```diff
@@ -7,6 +7,9 @@
     "build": "tsc && vite build",
     "preview": "vite preview"
   },
+  "resolutions": {
+    "@ark-ui/react": "^0.14.0"
```

Comment:
Nice. So this verifies that we need to add this to docs as well, correct? That this must be added for downstream apps?

Reply:
I don't think we should add this to docs since it won't be an issue in the future, and the "proper" fix is to set a peer dependency in the UI library anyways.